explorer/graphql/app/schema.graphql

```diff
@@ -66,6 +66,15 @@ type XMsg {
   "Hash of the source chain block"
   BlockHash: Bytes32!
 
+  "Hash of destination chain receipt"
+  ReceiptHash: Bytes32
+
+  "Status of the message"
+  Status: String!
```

Comment:
@DanFlannel will you add failure reasons later?

Reply:
Yes, that would be different than the status enum, I'll add a nullable field for failure reason